README.md

```diff
@@ -40,7 +40,7 @@
 - [Installation](#installation)
 - [Getting started](#getting-started)
 - [Usage](#usage)
-- [Licence](#licence)
+- [License](#license)
```

Comment:
This line is not a subject to change. We fixed it in this repository, didn't mirror to the main.

Reply:
got it, fixed